app/move/app/new/controllers/save.test.js

```diff
@@ -46,11 +42,17 @@ const mockValues = {
   to_location: 'Court',
   from_location: 'Prison',
   person: {
+    id: '__person_1__',
     first_names: 'Steve',
     last_name: 'Smith',
   },
   profile: {
-    id: '__profile',
+    id: '__profile_3__',
+    person: {
+      id: '__person_1__',
```

Comment:
Small thing, but could the profile id and the person id be similar so its more obvious they're connected?

Reply:
Not that they're used anywhere else, so this is probably not a useful change

Reply:
`person.id` and `profile.person.id` are the same, aren't they? `profile.id` is a different beast entirely…

Reply:
This was to show some scenarios where the profile may exist, but the person on that profile can be different (ie a person goes back a step and selects a different person).

>person.id and profile.person.id are the same, aren't they?

Not necessarily in all cases. That's what this also fixes.